app/controllers/houses_controller.rb

```diff
@@ -26,11 +26,10 @@ def update
     @house = House.find(params[:id])
     old_house_name = @house.house_name
     respond_to do |format|
-      if @house.update(house_params) && mod_params[:reason].present?
+      if @house.update(house_params)
         log_update(old_house_name)
         format.html { redirect_to '/houses' }
       else
-        @house.errors.add(:base, 'Must specify a reason.') if mod_params[:reason].blank?
         format.html { render :edit }
       end
```

Comment:
Respond to ... format is redundant if the only format you support is HTML. You can remove remove the `respond_to` and `format.html` blocks entirely and just put the `render` directly in the method body.